deploy/kubernetes/manifests/controller.yaml

```diff
@@ -20,20 +20,18 @@ spec:
       containers:
         - name: csi-external-attacher
           imagePullPolicy: Always
```

Comment:
Should probably use the default instead of Always

Reply:
Done